### PR TITLE
Docs update according to PR#1427

### PR DESCRIPTION
**Pull request by VisualGadget**
https://github.com/Koenkk/zigbee-herdsman-converters/pull/1427

### CODE REVIEW

File: docs/devices/ZNCLDJ12LM.md [hunk 1]
```diff
@@ -36,6 +36,29 @@ You can send a subset of options, all options that won't be specified will be re
...
+```yaml
+- alias: Calibrate curtain
+  trigger:
+  - platform: homeassistant
+    event: start
+  action:
+  - service: mqtt.publish
+    data:
+      topic: zigbee2mqtt/<FRIENDLY_NAME>/set
+      payload: "{'discovery': true}"
```
Comment (Koenkk) [inline docs/devices/ZNCLDJ12LM.md:53]: `{ 'discovery': ''}` should be good enough

File: docs/devices/ZNCLDJ12LM.md [hunk 2]
```diff
@@ -36,6 +36,29 @@ You can send a subset of options, all options that won't be specified will be re
 
 After changing `reverse_direction` you will need to fully open and fully close the curtain so the motor will re-detect edges. `reverse_direction` will get new state only after this recalibration.
 
+## Xiaomi ZNCLDJ12LM with battery
+
+| Picture | ![Xiaomi ZNCLDJ12LM with battery](../images/devices/ZNCLDJ12LM_w_battery.jpg) |
+
+If motor is used without battery it loses configuration when power down. After that you need to perform end stops calibration again. You can use the following automation for that:
```
Comment (Koenkk) [inline docs/devices/ZNCLDJ12LM.md:43]: Can you also provide an example with raw MQTT commands? (as not everybody might be running Home Assistant)

File: docs/devices/ZNCLDJ12LM.md (hunk 1 above)
Comment (VisualGadget) [reply docs/devices/ZNCLDJ12LM.md]: Ok. Done.

File: docs/devices/ZNCLDJ12LM.md (hunk 2 above)
Comment (VisualGadget) [reply docs/devices/ZNCLDJ12LM.md]: Done, please check. It it the expected format?

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**